tests/Serilog.Sinks.RabbitMQ.Tests/Approval/Serilog.Sinks.RabbitMQ.approved.txt

```diff
@@ -30,8 +30,8 @@ namespace Serilog
         public static Serilog.LoggerConfiguration RabbitMQ(
                     this Serilog.Configuration.LoggerSinkConfiguration loggerConfiguration,
                     string[] hostnames,
-                    string? username,
-                    string? password,
+                    string username,
```

Comment:
User name and password can be null in case of Certificate authentication.

Reply:
`ValidateRabbitMQClientConfiguration` throws in case `Username` or `Password` is null.

Reply:
That's wrong, I probably forgot to refactor this when I added full SSL support.

Reply:
To test this behavior the user name and password should not be set in the fixture that returns the Ssl configuration.

```csharp
            Username = UserName,
            Password = Password,
```

https://github.com/ArieGato/serilog-sinks-rabbitmq/blob/ecf7b1930355f6a1a34eedd10c323f50560f37fb/tests/Serilog.Sinks.RabbitMQ.Tests.Integration/RabbitMQFixture.cs#L75

Reply:
Do you expect some action here from me?

Reply:
No, I'll create an issue for this.